### PR TITLE
Add support for Polly presigning

### PR DESCRIPTION
**Pull request by dagnir**
## Description
Supports presigning for SynthesizeSpeechRequest.

## Motivation and Context
Part of #203 

## Testing
Unit and integration tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] Local run of `mvn install` succeeds
- [x] My code follows the code style of this project
- [x] My change requires a change to the Javadoc documentation
- [x] I have updated the Javadoc documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] A short description of the change has been added to the **CHANGELOG**
- [ ] My change is to implement 1.11 parity feature and I have updated [LaunchChangelog](https://github.com/aws/aws-sdk-java-v2/blob/master/docs/LaunchChangelog.md)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

File: services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+ * {@code
+ *     // Create a PollyPresigner using the default region and credentials.
+ *     // This is usually done at application startup, because creating a presigner can be expensive.
+ *     PollyPresigner presigner = PollyPresigner.create();
+ *
+ *     // Create a SynthesizeSpeechRequest to be pre-signed
+ *     SynthesizeSpeechRequest synthesizeSpeechRequest =
+ *             SynthesizeSpeechRequest.builder()
+ *                                    .text("Hello Polly!")
+ *                                    .voiceId("Salli")
```
Comment (millems) [inline services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java]: Enum?

File: services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ *                                    .outputFormat("PCM")
```
Comment (millems) [inline services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java]: Enum?

File: services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java (hunk 1 above)
Comment (dagnir) [reply services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java]: Fixed

File: services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java (hunk 2 above)
Comment (dagnir) [reply services/polly/src/main/java/software/amazon/awssdk/services/polly/presigner/PollyPresigner.java]: Fixed

**Pull request merged**